Lib/test/test_capi/test_misc.py

```diff
@@ -183,6 +184,12 @@ def test_docstring_signature_parsing(self):
         self.assertEqual(_testcapi.docstring_with_signature_and_extra_newlines.__text_signature__,
             "($module, /, parameter)")
 
+    @unittest.skipIf(MISSING_C_DOCSTRINGS,
+                     "Signature information for builtins requires docstrings")
+    def test_signature_parsing_with_defaults(self):
```

Comment:
Is it the right place for these tests? The tests above do not test `inspect.signature()`, they test `__text_signature__` and `__doc__` descriptors.

There are tests that use this class in `test_pydoc`, but they test `pydoc` code. I think that it is better to add this test in `test_inspect`.